reference-data-proxy/src/cronJobs/eStoreCronJobManager.ts

```diff
@@ -12,7 +13,7 @@ export const eStoreCronJobManager = new CronJobManager(
     onComplete: async () => {
       const collection = await getCollection('cron-job-logs');
       await collection.insertOne({ status: 'eStore Cron Job Manager started successfully', timestamp: new Date() });
-      console.info('eStore Cron Job Manager started successfully');
+      console.info('eStore Cron Job Manager started successfully at', new Date());
```

Comment:
Can we use `cronJobTimer` instead of `new Date()` to avoid instantiating another variable in the memory?

Reply:
good shout - I've updated that now